tests/system/datasources_test.go

```diff
@@ -0,0 +1,57 @@
+package system
+
+import (
+	"testing"
+)
+
+func TestDataSources(t *testing.T) {
+	t.Parallel()
+
+	s := NewSystemTest(t, "cases/datasources", nil)
+	s.Terraform.Apply()
+	defer s.Terraform.Destroy()
+
+	// Compare outputs of data and resource values (resource
+	// values have the '_expected' suffix)
+	checkOutput := func(key string) {
+		log.Debugf("Checking data source vs resource output for key: %s", key)
+		datasourceOutput := s.Terraform.Output(key)
+		resourceOutput := s.Terraform.Output(key + "_expected")
+
+		if datasourceOutput != resourceOutput {
+			log.Fatalf(
```

Comment:
Don't use `log` during tests to handle errors - it's just used for giving output while the test is running.

Use `t` to handle errors: `t.Error()`, `t.Fatal()`, etc (see: https://golang.org/pkg/testing/#T).